src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  margin-bottom: 46px;
+  margin-right: 48px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $c-text-main;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $c-text-second;
+    font-weight: 400;
+
+  }
+
+  &__review-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    color: $c-text-main;
+    font-weight: 400;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    line-height: 18px;
+  }
+
+  &__price-label {
+    color: $c-text-second;
+    font-weight: 400;
+  }
+
+  &__price {
+    font-size: 16px;
+    color: $c-text-main;
+    font-weight: 700;
+  }
+
+  &__buy-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
```

Comment:
To my opinion it's not good to set fixed height, we can reach needed sizes with paddings, without limiting our height. But in this case it's not critical.

Reply:
it's height of the button, let it be, later I can use the button as an independent block
btw: in my opinion is better to use in instead to, you can check it 😉